### ISSUE TITLE
Is there a way I can give a different batch size in each iteration into a fixed batch size in the feed dict?

### ISSUE THREAD

**Issue by emrullahpolat** [tensorflow/tensorflow#53723]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution: Windows 10 Pro
- TensorFlow installed from (source or binary)
- TensorFlow version : 2.6.0 
- Python version: 3.6.13
- CUDA/cuDNN version: 11.2
- GPU model: GeForce RTX 2080 Super

Hi, 
I am working on real time project using alexnet model. I am constantly getting frames at a constant FPS value.I need to get a prediction result for each frame. But I cut the objects whose number is not certain in each frame and give them to the trained model in a batch. Like the screenshot below;

![woring_correctly_loop_2](https://user-images.githubusercontent.com/40995578/148951684-1a84eb42-6710-4465-b82c-e528b1b2b421.png)

GPU **infrence time** works correctly when there is the same number of objects in each frame, as in the screenshot above. but when there is a different number of objects in each frame, the GPU infrence time returns with too much time. 
As far as I understand, parallel processes set themselves for different batch sizes. I can show you a screenshot of it below ;

![error_loop_log_image](https://user-images.githubusercontent.com/40995578/148952864-b120e85c-5a7e-40f0-8390-2c29f89b1e6e.png)

Between 0.005ms and 0.040 ms is a normal time for me but  values ​​other than these are not normal.

 ```
  input_batch = tf.compat.v1.placeholder(dtype=tf.float32, shape=[None, 227, 227, 1], name='input__batch')
```

 ```
  test_acc_step= sess.run('import/prediction_argmax_model:0', feed_dict={'import/x:0': input_static_batch_size_images})
```
With this usage, I am sending a **static batch** into the **session feed dict**.

**Can I give this to static size instead of None and feed it with dynamic batch in each iteration?** Or is there another way to do this?

**Comment by mohantym** [comment]
Hi @emrullahpolat ! Attaching relevant [thread](https://stackoverflow.com/questions/50551199/tensorflow-dataset-with-changing-batch-size-to-compute-test-loss-during-training) for reference.
Please open this issue in Tf discussion [forum ](https://discuss.tensorflow.org/)as there is a larger community there. Thank you!
